doc/source/contributing/how_to_contribute.rst

```diff
@@ -0,0 +1,81 @@
+.. _dev_guide_label: 
+
+How to contribute
+=================
+
+On this page we briefly provide basic instructions for contributing code to icepyx. 
+For contribution examples, please see contribution guidelines.
+In all cases, please follow the guidelines below. 
+
+1. Sign up for a GitHub account
+
+2. Clone the icepyx repo
+
+3. Create a new temporary branch off development 
+
+4. Add your name (in alphabetical order) to the contributors list
```

Comment:
Wanted to note that we'll need to update this now that we have the all-contributors bot. I'm happy to do that.